index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header-nav">
+        <h3>SheMer</h3>
+        <nav>
+            <ul>
+                <li><a href="/">Home</a></li>
+                <li><a href="#">About</a></li>
+                <li><a href="#">Contact</a></li>
+            </ul>
+        </nav>
+    </header>
+
+    <!--- Banner Section-->
+    <section class="bgimage" id="banner">
+
+        <div class="welcome-msg">
+            <h2 class="name">Hey We are SherMer</h2>
+            <p class="role">a web developer</p>
+        </div>
+
+    </section>
+    <!----Project Cards-->
+    <main class="container">
+        <h2>These are some of my projects</h2>
+
+        <div class="row">
+            <div class="col-sm-6">
+                <div class="card">
+                    <img class="card-img-top" alt="cards" width="400"
+                        src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute.jpg"="Card image cap">
```

Comment:
I would kindly suggest you use external styling as opposed to inline. kindly style the width of the image on CSS file.

Reply:
- [ ] As good as this point is, you forgot to tell the student why you thought so, and this is one of the reasons for code review. To further draw awareness to industry and best practices but the best way to make them stick and not be repeated is when students know `WHY` they should not repeat or do pieces of stuff in specific ways. Kindly add your reason 👍

Reply:
Amended as requested thank you.